examples/simple.py

```diff
@@ -4,13 +4,14 @@
 
 from quantuminspire.sdk.circuit import Circuit
 from quantuminspire.util.api.remote_runtime import RemoteRuntime
+from quantuminspire.util.configuration import Settings
 
 with Circuit(platform_name="spin-2", program_name="prgm1") as c:
     k = c.init_kernel("new_kernel", 2)
     k.x(0)
     k.hadamard(1)
 
-runtime = RemoteRuntime()
```

Comment:
I would say that the runtime would automatically load the settings, without the user having to specify them

Reply:
I've instantiated it in the runtime itself